sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -640,11 +639,47 @@ void Scheduler::GraphBuilder::markModifiedIfWrite(MemObjRecord *Record,
   }
 }
 
+EmptyCommand *
+Scheduler::GraphBuilder::addEmptyCmd(Command *Cmd,
+                                     const std::vector<Requirement *> &Reqs,
+                                     const QueueImplPtr &Queue) {
+  EmptyCommand *EmptyCmd =
+      new EmptyCommand(Scheduler::getInstance().getDefaultHostQueue());
+
+  if (!EmptyCmd)
+    throw runtime_error("Out of host memory", PI_OUT_OF_HOST_MEMORY);
+
+  EmptyCmd->MIsBlockable = true;
+  EmptyCmd->MEnqueueStatus = EnqueueResultT::SyclEnqueueBlocked;
+  EmptyCmd->MBlockReason = Command::BlockReason::HostTask;
```

Comment:
Minor. Should set a proper reason here as this method is called from host accessor creation process as well as from host task creation process.

Reply:
This'' go in distinct patch as it's corner case.